permissions_engine/permissions.rego

```diff
@@ -9,7 +9,9 @@ open_datasets = ["open1", "open2"]
 registered_datasets = ["registered3"]
 
 controlled_access_list = {"user1": ["controlled4"],    # TODO - should use iss:sub for this rather than username
-                          "user2": ["controlled5"]}
+                          "user2": ["controlled5"],
+                          "user3": ["controlled6", "controlled7"],
```

Comment:
Actually, can we have user3 have access to (say) controlled4 user4 have access to controlled5, to make it clear that users authenticated with different keycloaks can still have access to the same data sets?

Reply:
Sure! Will do.